spec/cli/commands/run_spec.rb

```diff
@@ -146,7 +146,7 @@
       end
 
       it "must set the :output_format option using the path's file extension" do
-        expect(subject.options[:output_format]).to be(Ronin::Core::OutputFormats::JSON)
+        expect(subject.outputs[0][1]).to be(Ronin::Core::OutputFormats::JSON)
```

Comment:
Might be safer to also test `outputs.length` as well as `outputs[0][0]` and `outputs[0][1]`.